### DISCUSSION TITLE
start menu does not work in build  25236

### DISCUSSION THREAD

**Discussion by hanifnau092** [valinet/ExplorerPatcher#1276] [General]
start menu doesn't work in build  25236

**Comment by antoniu200** [comment]
First of all, it is important to know that Insider builds are not supported.
ExplorerPatcher uses a custom dxgi.dll to enable the customizations it can apply to it. Since every Windows 11 update changes the StartMenuExperienceHost (responsible for the start menu), ExplorerPatcher requires downloading Symbols for your specific version, process that happens after you install it.
Try to open the Start Menu a couple of times while being connected to the Internet then allow explorer.exe 5 minutes to download the Symbols. You should get a notification which confirms that Symbols were downloaded, if they were. Restart explorer.exe after that happens, then try opening Start Menu again.
